forch/forchestrator.py

```diff
@@ -91,60 +92,87 @@ def initialize(self):
+                    radius_ip, radius_port)
 
     def _process_static_device_placement(self):
-        device_info = self._config.get('static_device_info', {})
-        if 'static_device_placement' in device_info:
-            placement_file = os.path.join(
-                os.getenv('FAUCET_CONFIG_DIR'), device_info['static_device_placement'])
-            device_placement_info = yaml_proto(placement_file, DevicesState).device_mac_placements
-            for eth_src, device_placement in device_placement_info.items():
-                self.process_device_placement(eth_src, device_placement)
+        static_placement_file = self._config.get('orchestration', {}).get('static_device_placement')
+        if not static_placement_file:
+            return
+        placement_file = os.path.join(
+            os.getenv('FAUCET_CONFIG_DIR'), static_placement_file)
+        device_placement_info = yaml_proto(placement_file, DevicesState).device_mac_placements
+        for eth_src, device_placement in device_placement_info.items():
+            self.process_device_placement(eth_src, device_placement)
+
+    def _process_static_device_behavior(self):
+        static_behaviors_file = self._config.get('orchestration', {}).get('static_device_behavior')
+        if not static_behaviors_file:
+            return
+        static_behaviors_path = os.path.join(
+            os.getenv('FAUCET_CONFIG_DIR'), static_behaviors_file)
+        devices_state = faucetizer.load_devices_state(static_behaviors_path)
+        for mac, device_behavior in devices_state.device_mac_behaviors.items():
+            self.process_device_behavior(mac, device_behavior)
+
+    def _initialize_faucetizer(self):
+        if not self._config.get('orchestration', {}).get('run_faucetizer'):
+            return
+        structural_config_file = self._config.get('orchestration', {}).get(
+            'structural_config_file', _STRUCTURAL_CONFIG_DEFAULT)
+        structural_config_path = os.path.join(
+            os.getenv('FAUCET_CONFIG_DIR'), structural_config_file)
+        LOGGER.info('Loading structural config from %s', structural_config_path)
+        with open(structural_config_path) as file:
+            structural_config = yaml.safe_load(file)
+            self._faucetizer = faucetizer.Faucetizer(structural_config)
+
+        interval = self._config.get('orchestration', {}).get('faucetize_interval', 1)
+        dynamic_config_file = self._config.get('orchestration', {}).get(
+            'dynamic_config_file', _DYNAMIC_CONFIG_DEFAULT)
+        dynamic_config_path = os.path.join(os.getenv('FAUCET_CONFIG_DIR'), dynamic_config_file)
+        self._faucetize_scheduler = HeartbeatScheduler(interval)
+        self._faucetize_scheduler.add_callback(functools.partial(
+            faucetizer.update_structural_config, self._faucetizer, structural_config_path))
+        self._faucetize_scheduler.add_callback(functools.partial(
```

Comment:
does this trigger faucet to try and reload the file each time?  If so, a few things:
1) Make the default period longer. 1s seems overly agressive.
2) File a bug to make this do change detection or based off of inode watching like faucet does.  
This is fine for now, but want to put it in the queue for eventual cleanup.

Reply:
Done

Reply:
Changed to 60s